packages/server/src/oauth/token.test.ts

```diff
@@ -1481,6 +1481,24 @@ describe('OAuth2 Token', () => {
     expect(res.body.error_description).toBe('Failed to verify code - check your identity provider configuration');
   });
 
+  test('Too many requests', async () => {
+    (fetch as unknown as jest.Mock).mockImplementation(() => ({
+      status: 429,
+      son: () => ({ email }),
```

Comment:
```suggestion
      json: () => ({ email }),
```

Reply:
This line could probably also just be removed, since the test doesn't seem to be using it